src/cmd/terraform.go

```diff
@@ -403,6 +403,15 @@ func flattenPredicate(key string, value *opslevel.Predicate) string {
   }
 `
 	if value != nil {
+		if value.Type == "matches_regex" {
+			config = `
+  %s {
+    type = "%s"
+	  value = <<-EOT
```

Comment:
Spacing was weird so I had to add spaces to get it to look correct on the export.  Not sure if you know what could be causing that.